podium-gateway/src/main/webapp/app/account/register/register.component.html

```diff
@@ -182,8 +182,7 @@ <h1 jhiTranslate="register.title">Registration</h1>
                     <label  class="form-control-label" for="jobTitle" >Function</label>
                     <input type="text" class="form-control" [(ngModel)]="registerAccount.jobTitle" id="jobTitle"
                            #jobTitle="ngModel" name="jobTitle" required minlength="1"
-                           placeholder="{{'global.form.jobTitle.placeholder' | translate}}"
-                           pattern="^[_'.@A-Za-z]*$">
+                           placeholder="{{'global.form.jobTitle.placeholder' | translate}}">
```

Comment:
Currently no max length has been supplied in the specs. Might be required in the future.